content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/index.mdx

```diff
@@ -32,18 +32,20 @@ In order to successfully complete this guide, you need:
 
 </SwitchTech>
 
+> # See [System requirements](/reference/system-requirements) for exact version requirements.
+
 <SwitchTech technologies={['*', 'sqlserver']}>
 
 - [Node.js](https://nodejs.org/en/) installed on your machine
 - A [Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/?view=sql-server-ver15) database
-  - [Microsoft SQL Server on Linux for Docker](../../../../concepts/database-connectors/sql-server/sql-server-docker)
-  - [Microsoft SQL Server on Windows (local)](../../../../concepts/database-connectors/sql-server/sql-server-local)
+  - [Microsoft SQL Server on Linux for Docker](/concepts/database-connectors/sql-server/sql-server-docker)
+  - [Microsoft SQL Server on Windows (local)](/concepts/database-connectors/sql-server/sql-server-local)
```

Comment:
Renders weird: https://deploy-preview-2140--prisma2-docs.netlify.app/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-node-sqlserver
![image](https://user-images.githubusercontent.com/183673/133078277-e5b4fae8-f9fc-4b3f-ba73-80945d548e7e.png)
POssibly related to https://github.com/prisma/docs/pull/2140/files#r707256404

Reply:
It was a duplicate. Fixed now